### ISSUE TITLE
false positive: import foo.bar

### ISSUE THREAD

**Issue by MarcoGorelli** [MarcoGorelli/cython-lint#3]
example:
```
        'cimport foo.bar\n'
        'cdef bool quox():\n'
        '    foo.bar.ni()\n',
```
results in `foo.bar imported but unused`
